terminal/bundles/org.eclipse.terminal.view.ui/src/org/eclipse/terminal/view/ui/internal/UIPlugin.java

```diff
@@ -248,6 +249,12 @@ protected void initializeImageRegistry(ImageRegistry registry) {
 		ImageDescriptor disabledTerminalViewIcon = ImageDescriptor.createWithFlags(enabledInputFieldIcon,
 				SWT.IMAGE_DISABLE);
 		registry.put(ImageConsts.ACTION_NewTerminalView_Disabled, disabledTerminalViewIcon);
+		URL enabledClearAllIconUrl = bundle
```

Comment:
You might want to use

`org.eclipse.jface.resource.ImageDescriptor.createFromFile(Class<?>, String)`

Reply:
Now I'm not sure that exactly here it will the best choice, since we will solve the same `Bundle` 5 times, I'll keep this advice for the case of single URL resolution.